upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = mpack-1.0
```

Comment:
I would appreciate if the directory name matched the wrap name, `ludocode-mpack`.

Reply:
Sorry, I got confused, this is the directory name of what is inside the archive. I reverted it.